Updated 3-vocabularies-and-ontologies (#149)
elixir-europe-training/ELIXIR-TrP-FAIR-Converge: docs/units/interoperable-data/topics/3-vocabularies-and-ontologies.md

learning_outcomes:
  1:

>     outcome: Understanding what ontologies and vocabularies are, explain the difference between the two and describe their purpose

Reviewer: ```suggestion
    outcome: Understand what ontologies and vocabularies are, explain the difference between the two and describe their purpose
```

  2:

>     outcome: Understanding how ontologies help to organizing metadata, and what part they play in making metadata schemas more 

Reviewer: ```suggestion
    outcome: Understand how ontologies help to organise metadata and what part they play in making metadata schemas more 
```

>     outcome: Understanding how ontologies help to organizing metadata, and what part they play in making metadata schemas more 

Reviewer: Something missing after more?

  3:

>     outcome: Understand/analyze the use of ontologies in a repository

Reviewer: ```suggestion
    outcome: Understand/analyse the use of ontologies in a repository
```

Reviewer: Note that UK spelling is preferred

  4:
    outcome: Understand the use of/apply ontology lookup services to find relevant ontologies and vocabularies
  5:

>     outcome: Understand/analyze/evaluate the steps, challenges and solutions involved in creating an ontology or vocabulary

Reviewer: ```suggestion
    outcome: Understand/analyse/evaluate the steps, challenges and solutions involved in creating an ontology or vocabulary
```

--- 

## Overview

### Topic

> This lesson plan will help you better understand what ontologies and vocabularies are, how they relate to the FAIR principles, and why they are important for research. At the end of this lesson, participants will be able to define and distinguish ontologies and vocabularies, understand how they are used in repositories, explain their role in metadata organisation, use ontology lookup services, and evaluate the process of creating an ontology or vocabulary.

Reviewer: The first sentence is learner-oriented, while the rest is not. I would try not to mix the style of these sentences.
